valhalla/thor/bidirectional_astar.h

```diff
@@ -43,7 +43,7 @@ class BidirectionalAStar : public PathAlgorithm {
 public:
   /**
    * Constructor.
-   * @param config A config object of key, value pairs
+   * @param config   Config with std::string for key and data
```

Comment:
was the previous description confusing ?

Reply:
No, This is rebase's problem. Bring it back?

Reply:
I don't see any reason to change this line. so, I'd propose change it back